doc/tech/release_notes/release_process.md

```diff
@@ -16,7 +16,7 @@ It should be considered a living document until it is agreed and signed by all t
 <a name="releasemodel"></a>
 ## Release Model
 
-As a guiding principle, all the development occurs in “Master” branch. All the contribution for the milestone (especially M3) goes into ”Master”. After M4 (Proof Reading), at one point in the release candidate (RC) (for now maintain RC0) for Baldy and based on experience we shall increase or limit what get into the final release. To make this happen, branch out from “Master”, create a delivery branch (“Baldy”,”Hallasan”, “Baraque”, etc). Developers can continue their next release branch work in “Master”. At the end of each release, artifacts are “tagged” in GitHub according to the guideline and principles defined.
+As a guiding principle, all the development occurs in “Master” branch. All the contribution for the milestone (especially M3) goes into ”Master”. After M4 (Proof Reading), at one point in the release candidate (RC) (for now maintain RC0) for Baldy and based on experience we shall increase or limit what get into the final release. To make this happen, branch out from “Master”, create a delivery branch (“Baldy”, “Baraque”, etc). Developers can continue their next release branch work in “Master”. At the end of each release, artifacts are “tagged” in GitHub according to the guideline and principles defined.
```

Comment:
@prabhubalan since RC is used within CNTT to refer to something else, can you use another acronym? For example, RlC or a better acronym. Thanks

Reply:
@pgoyal01 - Thanks for your comments. For simplicity, lets maintain (Release Candidate) without any acronym to avoid any confusion. I will remove RC and maintain Release Candidate instead.